Makefile

```diff
@@ -33,4 +40,17 @@ test:
 	bundle exec htmlproofer _site --check-html --disable-external
 	bundle exec scss-lint _sass
 
+docker:
+	./_bin/ensure-sibling-repo.sh
+	cp -r ../disclosure-backend-static/build/* . # Copy over the built resources from the backend
+	command -v docker >/dev/null 2>&1 || { echo >&2 "${RED}ERROR: Install docker at https://docs.docker.com/get-docker/${CLR}"; exit 1; }
+	docker build -t odca-jekyll .
+	docker run -it --rm -p 4000:4000 \
+          -v "$(PWD)/_includes/:/app/_includes" \
+          -v "$(PWD)/_layouts/:/app/_layouts" \
+          -v "$(PWD)/_plugins/:/app/_plugins" \
+          -v "$(PWD)/assets/:/app/assets" \
+          -v "$(PWD)/src/:/app/src" \
```

Comment:
these are what get synced between your local directory and the container and is basically all the stuff that supports code reloading. If there are other places that house code that may need to trigger auto-reloading, let me know and I can add them. For stuff that changes the build (like adding a new dependency), you'll have to actually ctrl-C the `make docker` command and rerun it so it triggers a rebuild.